content/en/blog/2024/go-contrib-removal.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Abandoned Go Contrib modules need code owners or will be removed
+linkTitle: Go Contrib modules ownership
+date: 2024-05-27
+author: >-
+  [Fabrizio Ferri-Benedetti](https://github.com/theletterf) (Splunk)
+issue: 4542
+sig: SIG Go
+# prettier-ignore
+cSpell:ignore: aws Benedetti Fabrizio Ferri gopkg labstack macaron moduled otelaws otelecho otellambda otelmacaron otelmongo otelmux
+---
+
+The Go SIG will remove Contrib modules that lack code owners. For each module,
```

Comment:
I think it's worth clarifying exactly what removal means and if the code will keep working for people. This will likely be top of mind for anyone.

Reply:
Excellent point. @pellared What's the fate of the "removed" modules?

Reply:
Existing versions are still available (and not retracted). "Removed" means that no new versions will be published until a code owner is found.

Reply:
In that case, "removed" isn't a good term, ... unless that's actually what you have in mind in the long term.

Ah, based on https://github.com/open-telemetry/opentelemetry-go-contrib/issues/5542 for example, it seems that it is your intention to have a two-phased approach for modules w/o code owners:

- Deprecate
- Remove

If that's the case, then this post should reflect that. Also, to address the concerns raised by @cartermp, timeframes should be given for each  phase/stage.

Reply:
The code will be removed from the repositories. But published packages/releases will remain available.